src/storage/index.ts

```diff
@@ -0,0 +1,48 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import getObj from 'lodash.get'
+import setObj from 'lodash.set'
```

Comment:
How do we feel about not doing these and always doing a flat namespace? I think we want to avoid situations where we query for nested things from different places without shared parsing logic, but the keypath APIs encourage exactly that.